app/models/post.rb

```diff
@@ -4,13 +4,18 @@ class Post < ApplicationRecord
   has_many :likes
   belongs_to :user
 
+  validates :title, presence: true, length: { in: 2..50 }
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_validations_model_spec.md#:~:text=Title%20must%20not%20exceed%20250%20characters.), Please ensure that Title must not exceed 250 characters, not 50 characters like you did.